app/models/cloud_volume.rb

```diff
@@ -54,14 +54,6 @@ def self.create_volume(ems_id, options = {})
     tenant = options[:cloud_tenant]
 
     created_volume = klass.raw_create_volume(ext_management_system, options)
-
```

Comment:
Hi, this propably causes build failure in https://github.com/ManageIQ/manageiq-providers-openstack - https://travis-ci.org/ManageIQ/manageiq-providers-openstack/jobs/221978185